diff -r d2588e0a4a95 /usr/lib/python3.4/venv/
--- /usr/lib/python3.4/venv/__init__.py	Tue May 27 21:25:31 2014 +0300
+++ /usr/lib/python3.4/venv/__init__.py	Tue May 27 19:25:32 2014 +0000
@@ -212,7 +212,8 @@
             for suffix in ('python', 'python3'):
                 path = os.path.join(binpath, suffix)
                 if not os.path.exists(path):
-                    os.symlink(exename, path)
+                    src_path = os.path.join(binpath, exename)
+                    copier(src_path, path)
         else:
             subdir = 'DLLs'
             include = self.include_binary
